lib/ui/profile/Profile.dart

```diff
@@ -348,4 +438,196 @@ class Profile extends StatelessWidget {
       ),
     );
   }
+
+  List<ProfileListItemViewModel> _profileItems(
+    ProfileViewModel viewModel,
+    BuildContext context,
+  ) {
+    List<ProfileListItemViewModel> list = [];
```

Comment:
Maybe profileItemsList will be better name?

Reply:
With name like items is good to me, just do not call list as list.

Reply:
items is better for me

Reply:
done